README.md

```diff
@@ -81,10 +81,9 @@ You may always check the current status with command `info`.
 
 You can also run a performance benchmark with command
 ```
-> benchmark alice 1000
+> benchmark alice 10 100
```

Comment:
When we add an amount to the benchmark command the console of both parties is spammed with `Received balance update` messages, where is the benefit of having an amount here?

Reply:
It's how we did it in Erdstall and I think it's better for a demo, because you see that actually something is happening. Additionally, you can verify that the the micro transactions had an impact on your on-chain balance after you close the channel.